tutorials/jitsi-ubuntu-focal-fossa/index.mdx

```diff
@@ -8,13 +8,13 @@ content:
 categories:
   - instances
   - domains-and-dns
-tags: compute mediaserver media video-sharing conference-call Jitsi-Meet
+tags: media Jitsi-Meet videoconferencing
 dates:
   validation: 2024-06-11
   posted: 2022-11-17
 ---
 
-Jitsi Meet fully encrypted, open source video conferencing tool that does not require subscription or the need to create an account.
+[Jitsi Meet](https://jitsi.org/jitsi-meet/) is a fully encrypted, open-source video conferencing tool that does not require an account or subscription.
```

Comment:
It's better not to link externally at the top of the page to avoid the robot going to this page instead of reading ours

Reply:
```suggestion
Jitsi Meet is a fully encrypted, open-source video conferencing tool that does not require an account or subscription.
```

Reply:
As both you and Nene made suggestions on these paragraphs, I'm merging both your suggestions into one, editing it and rejecting the doubled comment. Thanks!